server.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const express = require('express');
+
+// app instance of express
+const app = express();
+
+// Middleware
+const cors = require('cors');
```

Comment:
Hi there!

You mentioned there were still some CORS errors.
Can we add in: `app.use(cors())` and see if that does the trick?

See here: [Enable All Cors](https://expressjs.com/en/resources/middleware/cors.html) 👍